requirements.in

```diff
@@ -1,4 +1,4 @@
-https://github.com/codecov/shared/archive/a0d0333196f1c9577243ff15c85515754849127c.tar.gz#egg=shared
+https://github.com/codecov/shared/archive/aba05efd27be6a91cfacf8267cf5f04bc60cce19.tar.gz#egg=shared
```

Comment:
```suggestion
https://github.com/codecov/shared/archive/6eb8d08c28c684c900900ef4b1c8b325ba7c3591.tar.gz#egg=shared
```

Reply:
oops this suggestion is outdated now, but I will leave the comment here as a reminder to grab latest before merging https://github.com/codecov/shared/commits/main/